Packs/MicrosoftExchangeOnPremise/Integrations/EWSv2/EWSv2.yml

```diff
@@ -122,6 +122,12 @@ configuration:
   type: 8
   advanced: true
   defaultvalue: 'false'
+- display: Skip unparsable emails during fetch incidents
```

Comment:
Since email parsing issues can arise from a few different reasons (not just encoding), I wanted to make the description as inclusive as possible. Further explanation is provided in the integrations README and in xsoar.pan.dev troubleshooting documentation.